### PR TITLE
Select undiscovered wallet on "add wallet" click

### LINKED ISSUE #1380: Style Add hidden Wallet button as disabled
Terciary button **_+ Add hidden Wallet_** needs to be disabled when Device disconnected.

In such case, please add tooltip on hover saying "Connect your Trezor to add hidden wallet"

Screen Shot 2020-03-25 at 12 33 35

### PR DESCRIPTION
**Pull request by slowbackspace**
resolves https://github.com/trezor/trezor-suite/issues/1380#issuecomment-610260794

implemented more user-friendly behaviour in "add wallet" btn.
If user has undiscovered wallet clicking the button will now select the wallet instead of just showing tooltip that he needs to discover wallet first

### CODE REVIEW

File: packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx [hunk 1]
```diff
@@ -13,32 +13,31 @@ interface Props {
...
-            >
+            <Tooltip enabled={!device.connected} placement="top" content={tooltipMsg}>
                 <Button
                     data-test="@switch-device/add-wallet-button"
                     variant="tertiary"
                     icon="PLUS"
-                    isDisabled={!device.connected || !hasAtLeastOneWallet}
-                    onClick={async () => addDeviceInstance(device)}
+                    isDisabled={!device.connected || (!hasAtLeastOneWallet && !undiscoveredWallet)}
+                    onClick={async () => {
```
Comment (mroz22) [inline packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx]: does async make sense here?

Comment (slowbackspace) [reply packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx]: afaik no, but I don't why it was there before. edit: looks like it si doing some magic?

File: packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/index.tsx [hunk 2]
```diff
@@ -246,6 +246,9 @@ const DeviceItem = (props: Props & WrappedComponentProps) => {
                             device={device}
                             instances={props.instances}
                             addDeviceInstance={addDeviceInstance}
+                            selectDeviceInstance={(device: Props['device']) =>
```
Comment (szymonlesisz) [inline packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/index.tsx]: should be: selectDeviceInstance={selectDeviceInstance}

File: packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx (hunk 1 above)
Comment (szymonlesisz) [reply packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx]: looks like its not neccessary, you can remove it

File: packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx [hunk 3]
```diff
@@ -13,32 +14,32 @@ interface Props {
...
-
-    let tooltipMsg: string | JSX.Element = '';
-    if (!hasAtLeastOneWallet) {
-        tooltipMsg = <Translation id="TR_TO_ACCESS_OTHER_WALLETS_DISCOVER" />;
-    }
-    if (!device.connected) {
-        tooltipMsg = <Translation id="TR_TO_ACCESS_OTHER_WALLETS" />;
-    }
-
+    const undiscoveredWallet = instances.find(d => !d.state);
```
Comment (szymonlesisz) [inline packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx]: undiscoveredWallet is not necessary
you can do it like this:
`
onClick={() => hasAtLeastOneWallet ? addDeviceInstance(device) : selectDeviceInstance(instances[0])}
`

**Pull request merged**